pom.xml

```diff
@@ -251,6 +251,12 @@
 				<version>${netty.version}</version>
 				<optional>true</optional>
 			</dependency>
+			<dependency>
+				<groupId>javax.servlet</groupId>
+				<artifactId>javax.servlet-api</artifactId>
+				<version>${javax.servlet-api.version}</version>
+				<optional>true</optional>
```

Comment:
Are you sure this dep is really optional? I feel like the `Cookie` class would fail to load and crash.